src/win32/gui.zig

```diff
@@ -704,8 +705,8 @@ fn getCellSize(text_format: *win32.IDWriteTextFormat) XY(i32) {
         if (hr < 0) fatalHr("GetMetrics", hr);
     }
     return .{
-        .x = @max(1, @as(i32, @intFromFloat(@floor(metrics.width)))),
-        .y = @max(1, @as(i32, @intFromFloat(@floor(metrics.height)))),
+        .x = @max(1, @as(i32, @intFromFloat(@ceil(metrics.width)))),
+        .y = @max(1, @as(i32, @intFromFloat(@ceil(metrics.height)))),
```

Comment:
Doesn't this create a gap between the cells again?

Reply:
Oh I see, ok fair enough